api/v1beta2/ibmvpccluster_types.go

```diff
@@ -87,6 +91,14 @@ type VPCLoadBalancerSpec struct {
 	// +optional
 	// ++kubebuilder:validation:UniqueItems=true
 	AdditionalListeners []AdditionalListenerSpec `json:"additionalListeners,omitempty"`
+
+	// backendPools defines the load balancer's backend pools.
+	// +optional
+	BackendPools []BackendPoolSpec `json:"backendPools,omitempty"`
```

Comment:
Whats the use case you have in mind to expose Backedpool,Do you think user should worry about it? Also what happens if user does not pass anything here?

Reply:
I currently have some default pools implemented in test code. I have not yet decided whether to leave them in, or force the user to have to define pools (no defaults).